src/Fabulous.XamarinForms/Xamarin.Forms.Core.Attributes.fs

```diff
@@ -50,6 +50,7 @@ module View =
 
 module Label =
     let Text = Attributes.defineBindable<string> Xamarin.Forms.Label.TextProperty
+    let Style = Attributes.defineBindable<Style> Xamarin.Forms.Label.StyleProperty
```

Comment:
The `StyleProperty` is actually coming from `Xamarin.Forms.NavigableElement`, you can put it in its own module `module NavigableElement`.
This way your extension can be applied to `IViewWidgetBuilder` and be available to all controls :)

Reply:
Thanks for the review. Updated following your suggestions :)